CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ include_directories(${TILEDB_INCLUDE_DIR})
 #librt
 find_library(LIBRT_LIBRARY rt)
 
+#librt
+find_library(LIBRT_LIBRARY rt)
+
+#pgsql driver and dbi libs
+find_package(libdbi REQUIRED)
+include_directories(${LIBDBI_INCLUDE_DIR})
```

Comment:
Let's make this an optional dependency - if LIBDBI_FOUND, then a preprocessor macro -DLIBDBI_FOUND should be passed (see add_definitions() for libcsv in the CMakeLists.txt).

In the source code,
```
#ifdef LIBDBI
  //All libdbi related code
#endif
```